compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Pretty.hs

```diff
@@ -635,6 +635,8 @@ pPrintTemplate lvl modName (Template mbLoc tpl param precond signatories observe
           ]
       implementsDoc = map (pPrintTemplateImplements lvl) (NM.toList implements)
 
+-- TODO(MA): Handle interface instances in interfaces
```

Comment:
Does this mean we still cannot use retroactive instances  ?

Reply:
This is perfectly fine we do that in another PR, but I do not clearly understood the title of the PR in this case.

Reply:
Note that this is only pretty printing. This doesn’t matter for actual retroactive instance support.

Reply:
thanks @cocreature

Reply:
:+1: